src/wizard/dotnet/wpf.md

```diff
@@ -0,0 +1,64 @@
+---
+name: WPF
+doc_link: https://docs.sentry.io/platforms/dotnet/guides/wpf/
+support_level: production
+type: language
+---
+
+## Install the NuGet package
+
+```shell
+# Using Package Manager
+Install-Package Sentry -Version {{ packages.version('sentry.dotnet') }}
+
+# Or using .NET Core CLI
+dotnet add package Sentry -v {{ packages.version('sentry.dotnet') }}
+```
+
+<div class="alert alert-info" role="alert"><h5 class="no_toc">Using .NET Framework prior to 4.6.1?</h5>
+    <div class="alert-body content-flush-bottom">
+        <a href="https://docs.sentry.io/clients/csharp/">Our legacy SDK</a> supports .NET Framework as early as 3.5.
+    </div>
+</div>
+
+## Initialize the SDK
+
+Initialize the SDK as early as possible, like in the constructor of the `App`:
+
+```csharp
+using System.Windows;
+using Sentry;
+
+public partial class App : Application
+{
+    public App()
+    {
+        DispatcherUnhandledException += App_DispatcherUnhandledException;
+        SentrySdk.Init("___PUBLIC_DSN___");
+    }
+
+    void App_DispatcherUnhandledException(object sender, DispatcherUnhandledExceptionEventArgs e)
+    {
+        SentrySdk.CaptureException(e.Exception);
+
+        // If you want to avoid the application from crashing:
+        e.Handled = true;
+    }
+
+    protected override void OnExit(ExitEventArgs e)
+    {
+        SentrySdk.Close();
+    }
```

Comment:
Tested and the ProcessExit integration is able to flush so no need to close it here
```suggestion
```